.github/workflows/no-name.yml

```diff
@@ -0,0 +1,9 @@
+name: No name jobs
+on: push
+
+jobs:
+  danger_check_pr_ios_specific:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Hello world
+        run: echo "Hello 🌏👋"
```

Comment:
A workflow with jobs that have no name is rendered as "Workflow name / job id"

<img width="907" alt="Screen Shot 2020-02-12 at 10 37 35" src="https://user-images.githubusercontent.com/1218433/74289692-d6eafb80-4d83-11ea-88e0-47260f6e0ecc.png">